packages/cli/commands/init.js

```diff
@@ -189,6 +209,10 @@ exports.builder = yargs => {
       describe: i18n(`${i18nKey}.options.account.describe`),
       type: 'string',
     },
+    useHiddenConfig: {
+      describe: i18n(`${i18nKey}.options.useHiddenConfig.describe`),
+      type: 'boolean',
+    },
```

Comment:
Should we add a `.conflicts('config', 'useHiddenConfig')` since they are at odds with each other?

Reply:
That sounds great! I'm manually checking for the conflict right now, but this is so much easier 🥳

Reply:
Addressed in https://github.com/HubSpot/hubspot-cli/pull/1177/commits/b5d40ded978b7c7d0858448a30cd65b2badd1448